workshop_R.Rmd

```diff
@@ -618,6 +633,11 @@ This means that while the code `readr::read_csv(insert_filename/filepath_here)`
 
 `chosen_name <- readr::read_csv(insert_filename/filepath_here)`
 
+When you're reading in the HEA CSV file for the below example, make sure that you have saved it inside a folder called "data", and that the file name matches exactly (e.g. doesn't have (1) or similar on the end).
```

Comment:
Might need to give the extra step that people might need to create the data folder themselves? I was considering whether we just remove all the 'data/' stuff and don't mention it but I think it's good for people to see how file paths work within projects (it's all relevant to the .rproj file)

Reply:
Done!